src/libprojectM/PresetLoader.cpp

```diff
@@ -149,6 +149,24 @@ const std::string & PresetLoader::getPresetName ( PresetIndex index ) const
 	return _presetNames[index];
 }
 
+
+// Get vector of preset names
+const std::vector<std::string> &PresetLoader::getPresetNames() const
+{
+	return _presetNames;
+}
+
+// Get the preset index given a name
+const unsigned int PresetLoader::getPresetIndex(std::string &name) const
+{
+	//Need a better structure than a vector for reverse lookup
+	for (int index = 0; index < _presetNames.size(); index++)
```

Comment:
is there an `indexOf()` method or something like that in Vector?

Reply:
done